fireblocks_sdk/sdk.py

```diff
@@ -47,6 +47,70 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
         self.token_provider = SdkTokenProvider(private_key, api_key)
         self.timeout = timeout
 
+    def get_nft(self, token_id: str):
+        url = "/v1/nfts/tokens/" + token_id
+
+        return self._get_request(url)
+
+    def get_nfts(self, token_ids: str):
```

Comment:
Let's decide for a string array or string concatenation string first 🙏

Reply:
Done